nautobot/circuits/filters.py

```diff
@@ -55,7 +55,7 @@ class ProviderFilterSet(NautobotFilterSet):
         label="Site (ID or slug)",
     )
     location = TreeNodeMultipleChoiceFilter(
-        field_name="circuits__terminations__location__slug",
+        field_name="circuits__terminations__location",
```

Comment:
Are there any other cases of this incorrect pattern in the code base? Or were these the only two?

Reply:
These are the only two as far as I can tell.